Equilobe.DailyReport.SL/BitBucketService.cs

```diff
@@ -22,16 +46,17 @@ public Log GetLog(ISourceControlContext context)
             return BitBucketLogHelper.LoadLog(commits, pullRequests, context.FromDate);
         }
 
-        public List<PullRequest> GetAllPullRequests(SourceControlOptions sourceControlOptions)
+        public List<PullRequest> GetAllPullRequests(SourceControlOptions sourceControlOptions, DateTime? lastSync = null)
```

Comment:
Better we should rename this. This does not get all pull requests necessarily. Just prs since last sync